src/components/ConnectForm.vue

```diff
@@ -2,6 +2,7 @@
 	<div class="connectPage">
 		<Logo />
 		<div class="connectContainer">
+			<div class="helpBtn" @click="showHelp" title="Show help"><i class="fas fa-question fa-fw"></i> Help</div>
```

Comment:
Should we give this a better look and feel? Right now it looks different to any other thing you can click (i.e. button or link)

Reply:
I just copied the snippet from IDE.vue, that's why it looks like it does. And I thought it didn't look too bad, but you're right, in this context the appearance is much different from all other clickable things. I don't see a strong need for changing the look&feel but am open for suggestions as I see your point.

I tried a button but don't really like it that much:
![grafik](https://user-images.githubusercontent.com/17781682/132021519-2f668645-2d3c-4dda-b85e-2230d6d2cae6.png)

Reply:
IMO a link would be misleading because then I'd expect a new webpage to open (documentation or something), not a tour window.

Reply:
Hmm, I'm thinking maybe we should just make it red (often help is red) or use the blueish link color instead of black, which is the color for non-interactive elements/text.

Reply:
By the way, how does the help button behave with longer titles (usually in the login screen, e.g. for GEE or VITO)?

Reply:
I think red is too intrusive.

Blue like this?
![grafik](https://user-images.githubusercontent.com/17781682/132023189-1e1b32cc-8c1d-4053-8607-c5e29e1a4995.png)

I also included the answer to the second comment into this screenshot.

Reply:
I'd think red is okay as it should be intrusive. If people look for help, it should stand out and be easy to find.

But the blue also works for now. Did you use the $linkColor variable from theme.scss for it?

Reply:
I used `$mainColor` because I didn't know of the other and it seemed to be what I wanted, but yeah I can switch that easily.

The standing out is all good and such, but still it just destroys the design IMO :D

![grafik](https://user-images.githubusercontent.com/17781682/132024096-11198d5f-811d-46cc-981d-8250981a07bf.png)

Quick sketch of your idea, doesn't convince me TBH:

![grafik](https://user-images.githubusercontent.com/17781682/132026114-a843a18a-ad0a-4e87-9d44-029d91dd3b1a.png)

Now I start liking the button from earlier :thinking: 

![grafik](https://user-images.githubusercontent.com/17781682/132025293-d22a7679-97a4-4a70-8c94-423f029dd630.png)

Reply:
The button looks weird. I like the round button, but the colors don't fully match. Maybe #b61665 or #9D3F22 would work. But anyway we can also just use the blue link color for now.